### PR TITLE
Stop using vararg for formatting

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**
API affecting part of https://github.com/sociomantic-tsunami/ocean/pull/187

### CODE REVIEW

**Review by leandro-lucarella-sociomantic — APPROVED** [review]
LGTM but not to travis.

**Comment by mathias-lang-sociomantic** [comment]
I think it's because it need #187 first :)

**Comment by mihails-strasuns-sociomantic** [comment]
Rebased

File: relnotes/vararg.migration.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* `ocean.io.console.AppStatus`
+
+  `formatStaticLine` method was changed to use template-based variadic argument
+  list instead of runtime one.
```
Comment (nemanja-boric-sociomantic) [inline relnotes/vararg.migration.md:4]: What kind of change this actually require from the user? It should be called in the same way, shouldn't it?

Comment (mihails-strasuns-sociomantic) [reply relnotes/vararg.migration.md:4]: If user code was passing in `__va*`, calling method will need to also be rewritten to use template instead. Direct calls will keep working as-is.

Comment (nemanja-boric-sociomantic) [reply relnotes/vararg.migration.md:4]: I see :+1:

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Pull request merged**